src/getting-started/project-setup/using-cargo-web.md

```diff
@@ -27,6 +27,6 @@ cargo web start
 * `asmjs-unknown-emscripten`
 
 {% hint style="info" %}
-For `*-emscripten` targets, you'll need to install the Emscripten SDK
+For `*-emscripten` targets, `cargo-web` will automatically install the Emscripten SDK and generate targets for you.
```

Comment:
This sentence is different from the `zh_cn` branch, as `zh_cn` branch is more detailed, I just translate it into English.

Reply:
Ah thanks but actually this sentence is only correct for Linux, so I removed it. Can you please update the `zh_cn` to match the `master` branch?

Reply:
My pleasure.